Packages/cdms2/Lib/dataset.py

```diff
@@ -283,9 +283,10 @@ def openDataset(uri,mode='r',template=None,dods=1,dpath=None, hostObj=None):
     uri = string.strip(uri)
     (scheme,netloc,path,parameters,query,fragment)=urlparse.urlparse(uri)
     if scheme in ('','file'):
+        if uri[:7].lower() == "file://":
```

Comment:
@chaosphere2112 I'm not 100% sure how the `file://` got appended to the uri in the first place... I have a vague recollection of you asking me to do this forthe GUI so I'll take the blame. But it might be worth tracing down.